src/js/contentscripts/socialwidgets.js

```diff
@@ -443,8 +443,29 @@ function createReplacementWidget(widget, elToReplace, activationFn) {
 
   let textDiv = document.createElement('div');
   textDiv.style = styleAttrs.join(" !important;") + " !important";
-  textDiv.appendChild(document.createTextNode(
-    TRANSLATIONS.widget_placeholder_pb_has_replaced.replace("XXX", name)));
+  // add link to replaced widget text if it has a src
+  if (elToReplace.getAttribute("src")) {
+    let widget_link = document.createElement("a");
+    widget_link.rel = "noreferrer";
+    widget_link.target = "_blank";
+    widget_link.href = elToReplace.src;
+    widget_link.innerHTML = name + ' widget';
+
+    let widget_link_styles = [
+      "color: #ec9329",
+      "margin-left: 3px",
```

Comment:
Is this to fix white space being collapsed by the flex layout? We should instead create a wrapper div for the text and link elements:
```
textDiv
    wrapperDiv
        textNode
        widgetLink
        textNode
```